joschahenningsen/TUM-Live
# Show transcript for lectures with subtitles

### Motivation and Context
See #1369.

### Description
Updated web client to include the option to show lecture transcript with available subtitles.
When the transcript is enabled, the user can read/scroll/fast-forward through the text (see demo video below).


### Steps for Testing
Prerequisites:
- 1 Video with subtitles
- 1 Video without subtitles

1. Open a video.
2. a "Show transcript" button should appear if it has subtitles.
3. Now, you should be able to see the transcript next to the video (/ below the video if you're on a mobile device). 

### Screenshots
https://github.com/user-attachments/assets/912302be-2ae6-4f56-8c07-3760f80e0991
